src/cljs/main/broadfcui/page/workspace/summary/workspace_cloner.cljs

```diff
@@ -41,27 +41,31 @@
           (style/create-form-label "Description (optional)")
           (style/create-text-area {:style {:width "100%"} :rows 5 :ref "wsDescription"
                                    :defaultValue (:description props)})
-          (if (:is-protected? props)
-            [:div {} "Cloned workspace will automatically be protected because this workspace is protected."]
-            [comps/Checkbox
-             {:ref "protected-check"
-              :label "Workspace intended to contain NIH protected data"
-              :disabled? (not= (:protected-option @state) :enabled)
-              :disabled-text (case (:protected-option @state)
-                               :not-loaded "Account status has not finished loading."
-                               :not-available "This option is not available for your account."
-                               nil)}])
+          [:div {:style {:display "flex"}}
```

Comment:
**Before:** Showed the checkbox to protect the workspace with the dbGapAuthorizedUsers realm if the source workspace was unprotected, showed no textbox if the source had the realm already
**After:** Same logic as above but shows a drop down of all of your authorization domains instead of checkbox.